build.gradle

```diff
@@ -1342,6 +1348,14 @@ project(':clients') {
     implementation libs.lz4
     implementation libs.snappy
     implementation libs.slf4jApi
+    implementation libs.opentelemetryProto
+
+    // declare runtime libraries
```

Comment:
can we add a comment to explain which dependencies should be declared here?

Reply:
Done.